src/lib/select/select.scss

```diff
@@ -3,26 +3,27 @@
 @import '../core/style/form-common';
 @import '../core/a11y/a11y';
 
-$md-select-trigger-height: 30px !default;
+$md-select-trigger-height: 27px !default;
```

Comment:
We may have to think more about how we want to approach this- the md-input changes its size based on the current font size, so this may only look right with a specific size